x-pack/plugins/index_lifecycle_management/public/application/store/defaults/cold_phase.js

```diff
@@ -24,7 +25,14 @@ export const defaultColdPhase = {
   [PHASE_FREEZE_ENABLED]: false,
   [PHASE_INDEX_PRIORITY]: 0,
 };
-export const defaultEmptyColdPhase = {
-  ...defaultColdPhase,
+
+export const defaultColdPhaseWhenEditing = {
```

Comment:
I feel like this hard-coding obscures the differences between the "whenNew" values and the "whenEditing" values...

Reply:
I agree with separating the 2 default objects because I think they actually should not be as closely linked as they are now. For me, defaultWhenNew contains default values when creating a new policy. It has some "recommendations", like "index priority" set to 50 in warm phase, for example. 
The other object doesn't contain default settings in my understanding. It is only used when an existing policy is loaded from ES and initializes the values. For example, it sets "index priority" to an empty string as the input field needs some value as a prop. For delete phase, for example, it sets snapshot policy name to an empty string, because this field doesn't exist when the "wait for snapshot" action is not used.